typings/enums.d.ts

```diff
@@ -28,32 +28,18 @@ export enum ApplicationCommandPermissionTypes {
 }
 
 export enum ChannelType {
```

Comment:
Shouldn't this be called `ChannelTypes` to be consistent with other enums?

Reply:
There were two enums: ChannelType and ChannelTypes and I had to remove one so I went with that but it can be renamed yeah. 
While we’re at it, should I also change the only other enum with a singular name to plural (InviteTargetType)? @discordjs/the-big-3

Reply:
Addressed in the latest commit